### PR TITLE
Documentation around Pomerium/Istio/Grafana

### PR DESCRIPTION
**Pull request by bjoernw**
## Summary
I'm not sure if I put this in the right place but I wanted to add an example of how to get Pomerium to play nicely with Istio within the context of adding authentication to Grafana.

Let me know what you think, I'm happy to make changes.

**Checklist**:
- [X] updated docs
- [X] ready for review

### CODE REVIEW

**Comment by desimone** [comment]
\cc @travisgroth 's wheelhouse for sure

File: docs/configuration/examples.md [hunk 1]
```diff
@@ -129,3 +129,39 @@ Customize for your identity provider run `docker-compose up -f nginx.docker-comp
 [helloworld]: https://hub.docker.com/r/tutum/hello-world
 [httpbin]: https://httpbin.org/
 [https load balancing]: https://cloud.google.com/kubernetes-engine/docs/concepts/ingress
+
+## Istio
```
Comment (desimone) [inline docs/configuration/examples.md:133]: nit: do you mind putting links to the referenced products under the header (e.g. Istio, Certmanager, and Grafana?)

File: docs/configuration/examples.md [hunk 2]
```diff
@@ -129,3 +129,39 @@ Customize for your identity provider run `docker-compose up -f nginx.docker-comp
...
+
+- Istio provides mutual TLS via sidecars and to make Istio play well with Pomerium we need to disable TLS on the Pomerium side.
+- We need to provide Istio with information on how to route requests via Pomerium to their destinations.
+- The following example shows how to make Grafana's auth proxy work with Pomerium inside of an Istio mesh.
+
+#### Gateway
+
+We are using the standard istio-ingressgateway that comes configured with Istio and attach a Gateway to it that deals with a subset of our ingress traffic based on the Host header (in this case `*.yourcompany.com`). This is the Gateway to which we will later attach VirtualServices for more granular routing decisions. Along with the Gateway, because we care about TLS, we are using Certmanager to provision a self-signed certificate (We are skipping the Certmanager setup here).
```
Comment (desimone) [inline docs/configuration/examples.md:141]: nit: Capitalized W in the middle of a sentence

> ("**W**e are skipping

**Comment by travisgroth** [comment]
LGTM

**Pull request merged**